test/test_macho.rb

```diff
@@ -313,8 +321,15 @@ def test_change_install_name
 
       file.write(actual)
 
-      # compare actual and expected file hashes, to ensure file correctness
-      assert equal_sha1_hashes(actual, expected)
```

Comment:
I think you misunderstood me. I'm perfectly fine with a hash comparison and it being treated equally with bit-for-bit equality. What I was referring to, was the fact that the hash comparison is being removed here.